src/js/components/InputControl/InputControl.tsx

```diff
@@ -69,9 +69,12 @@ const InputControl = ({
         {...inputProps}
         checked={value} // has to come after spreading input props to support indeterminate
       />
-      <label className="Checkbox-label" htmlFor={name} {...labelProps}>
-        {label}
-      </label>
+      <label
+        dangerouslySetInnerHTML={{ __html: label as string }}
```

Comment:
Why not just allow the label to be either a string or a react node?

Reply:
@bradcypert I've refactored the code so that we aren't using `dangerouslySetInnerHTML`